src/app/static/src/js/components/publishReports/publishReports.vue

```diff
@@ -17,6 +28,22 @@
     export default {
         name: "publishReports",
         components: {DateGroup},
-        props: ["reportsWithDrafts"]
+        props: ["reportsWithDrafts"],
+        data() {
+            return {
+                expand: 0,
+                collapse: 0
+            }
+        },
+        methods: {
+            expandChangelogs(e) {
+                e.preventDefault();
+                this.expand = this.expand + 1;
```

Comment:
This was the neatest way I could think of transmitting event data from the parent component down to the child - just increment a prop value when the button is clicked, and watch for changes in the child. What do you think?